common/app/views/emailEmbed.scala.html

```diff
@@ -66,7 +66,7 @@
             }
 
             function onSubmit(e) {
-                e.preventDefault()
+                e.preventDefault();
```

Comment:
```suggestion
                e.preventDefault()
```

Reply:
I thought this too! This is a bug which @dblatcher caught, where without the `;` the javascript engine tries to run the next line (an IIFE) as part of the same statement as `e.preventDefault()` so you end up with `e.preventDefault()(/* iife */)`, giving an error!

A wider problem here is that there's no linting available in these files :(

Reply:
ohhh nice catch